org/org_management.py

```diff
@@ -186,6 +201,56 @@ def _validate_github_org_cfg(cfg):
         jsonschema.validate(cfg, OrgGenerator._GITHUB_ORG_CFG_SCHEMA)
         return cfg
 
+    # https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0005-github-teams-and-access.md
+    @staticmethod
+    def _generate_wg_teams(wg) -> Tuple[str, Dict[str, Any]]:
+        name = OrgGenerator._kebab_case(f"wg-{wg['name']}")
+        maintainers = {u["github"] for u in wg["execution_leads"]}
+        maintainers |= {u["github"] for u in wg["technical_leads"]}
+        approvers = {u["github"] for a in wg["areas"] for u in a["approvers"]}
+        repositories = {r for a in wg["areas"] for r in a["repositories"] if r.startswith("cloudfoundry/")}
```

Comment:
We will have to rework this line later when we onboard other orgs into the automation but for now it should be fine.